### PR TITLE
Moves restartless extension links out from the source files folder.

### PR DESCRIPTION
**Pull request by yurydelendik**

### CODE REVIEW

File: extension/firefox/Makefile [hunk 1]
```diff
@@ -53,5 +55,24 @@ ensureoutputdir: clean
 clean:
 	-rm -rf build
 	-rm -rf $(BUILD_DIR)
+	-rm -rf $(RESTARTLESS_DIR)
 
-PHONY: build clean ensureoutputdir
+restartless:
+	-rm -rf $(RESTARTLESS_DIR)
```
Comment (tobytailor) [inline extension/firefox/Makefile]: Is this a typo?

Comment (yurydelendik) [inline extension/firefox/Makefile]: first minus? no, it's just ignoring the error code

**Comment by yurydelendik** [comment]
btw, the links created by the previous version in the extension/firefox folder must be removed

**Comment by tobytailor** [comment]
r+

**Pull request merged**